libxmp-config-version.cmake.autotools

```diff
@@ -0,0 +1,21 @@
+set(PACKAGE_VERSION "4.5.1")
```

Comment:
Don't forget to bump this string for new releases.
I hard coded it here, as is done for `libxmp.pc`.

Reply:
OK. And these are all great, thanks!

Will merge after @Wohlstand tests too.